app/models/alchemy/essence_node.rb

```diff
@@ -16,16 +14,5 @@ class EssenceNode < BaseRecord
     )
 
     delegate :name, to: :node, prefix: true, allow_nil: true
-
-    def ingredient=(node)
-      case node
-      when NODE_ID
-        self.node = Alchemy::Node.new(id: node)
-      when Alchemy::Node
-        self.node = node
-      else
-        super
-      end
-    end
```

Comment:
Would you mind to extract that into a separate PR, please? We might need to backport it.

Reply:
done, see #1834